### ISSUE TITLE
Outdated RAVEN code interface docs

### ISSUE THREAD

**Issue by AaronEpiney** [idaholab/raven#666]
--------
Issue Description
--------
The docs have not been updated fro the conversion module in the RrR code interface. See test test_raven_running_raven_int_models.xml

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with a type: defect or improvement?
- [x] 2. Is it tagged with a priority: critical, normal or minor?
- [x] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [x] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [x] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed?
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] 4. If the issue is a defect, does it impact the latest stable branch? If yes, is there any issue tagged with stable (create if needed)?
- [x] 5. If the issue is being closed without a merge request, has an explanation of why it is being closed been provided?

**Comment by PaulTalbot-INL** [comment]
Proposed email for PR merge:

Hello, RAVEN users!

As of merge #668, RAVEN's "devel" branch has received a couple notable updates.

First off, we understand the install and update process has been cumbersome, and we've worked to streamline it.

Now, to update RAVEN, the following actions are consistent across all platforms.  From the "raven" folder (~/projects/raven might be different depending on where you installed it):

- git checkout devel
- git pull
- ./scripts/establish_conda_env.sh --install
- ./build_raven

That's it!  Setting up the python environment is done entirely through "establish_conda_env.sh" and make is rolled into "build_raven".  The appropriate libraries will be installed through conda to be current with the branch of RAVEN that has been checked out.

Secondly, we have updated several libraries to newer versions, including plotting tool matplotlib as well as numerical tools scipy and numpy.  This significantly sped up some processes on Windows and keeps us up to date with critical changes.


As always, let us know if you run into any issues!
